Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -49,8 +49,25 @@
 # OptionalTupleBits - This member stores bits of whether each following member is serialized. Attribute must be on first member.
 # OptionalTupleBit - The name of the bit indicating whether this member is serialized.
 
+
+class Template(object):
+    def __init__(self, template_type, namespace, name, enum_storage=None):
```

Comment:
We'll likely need condition here at some point.  Maybe also just for certain instantiations.  Can be done later.